tmt/steps/provision/podman.py

```diff
@@ -338,6 +369,19 @@ def remove(self) -> None:
             self.podman(Command('container', 'rm', '-f', self.container))
             self.info('container', 'removed', 'green')
 
+        if self.network:
+            # will remove the network if there are no more containers attached to it
```

Comment:
```suggestion
            # Will remove the network if there are no more containers attached to it
```

Just consistent style.